src/components/form/button.js

```diff
@@ -73,7 +73,7 @@ const SharedButtonStyle = css`
             `
         if (props.white)
             return css`
-                border: 2px solid var(--color-white);
+                border: 2px solid var(--color-black-5);
```

Comment:
any reason why?

Reply:
It's from Zeplin - and as far as I can see this type of button is not used anywhere else so should be ok?